### ISSUE TITLE
[iPad Bug] UI is completely unresponsive after tapping outside of Files browser to dismiss modal

### ISSUE THREAD

**Issue by manderson-productions** [essej/sonobus#6]
I found an issue running Sonobus 1.0.3 (32) on iPad Pro (12.9-inch) (3rd generation) running iOS 14.0

1. Open Sonobus on iPad
2. Tap on the Files button (next to the Record button)
3. Tap outside of the modal in order to dismiss the Files browser (as opposed to tapping the Cancel button)

Expected: The UI is still responsive and I can still use the app as I normally would.
Actual: The UI becomes unresponsive and I can no longer interact with the app, causing me to force quit. The issue is likely that the modal is not resigning its first responder status.

**Comment by essej** [comment]
Yeah, I know about this one, it is a JUCE bug... I will be looking into it, thanks
